cmd/libsnap-confine-private/utils.c

```diff
@@ -261,3 +261,42 @@ bool sc_wait_for_file(const char *path, size_t timeout_sec)
 	}
 	return false;
 }
+
+const char *run_systemd_container = "/run/systemd/container";
+
+static bool _sc_is_in_container(const char *p)
+{
+	// see what systemd-detect-virt --container does in
+	// https://github.com/systemd/systemd/blob/5dcd6b1d55a1cfe247621d70f0e25d020de6e0ed/src/basic/virt.c#L749-L755
```

Comment:
Can you reference https://systemd.io/CONTAINER_INTERFACE/ which has a somewhat more stable URL and is higher-level.

Reply:
added